src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsEventSourceLifetime.cs

```diff
@@ -21,6 +21,7 @@ public class TestsEventSourceLifetime
         /// </summary>
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)] // non-Windows EventSources don't have lifetime
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "ExerciseEventSource use private member reflection")]
```

Comment:
Can you point at what private reflection happens here?  This is likely expected, and so we'll want to have an issue filed to track and fix it.